packages/elements/src/components/ino-segment-button/ino-segment-button.scss

```diff
@@ -1,96 +1,87 @@
 @use 'base/mdc-customize';
 @use '@material/button/mdc-button';
 @use '@material/button';
-@use 'base/theme';
+@use 'base/animation';
 @use 'base/typography';
+@use 'base/common';
+@use 'base/new-theme' as theme;
 
-// Borders
-$edged-border: 0;
-$left-segment-border: 0 0 0 24px;
-$right-segment-border: 0 24px 24px 0;
-$left-right-segment-border: 0 24px 24px 24px;
+/* borders */
+$round-border: 24px 24px 24px 24px;
+$edged-border: 24px 0 24px 24px;
 
-/**
- * @prop --ino-segment-button-color: Color of the segment button if inactive
- * @prop --ino-segment-button-checked-color: Color of the segment button if checked
- * @prop --ino-segment-button-hover-color: Color of the segment button on hover
- * @prop --ino-segment-button-disabled-color: Color of the segment button if disabled
- * @prop --ino-segment-button-hover-shadow: Color and shape of the shadow on hover
- * @prop --ino-segment-button-checked-shadow: Color and shape of the shadow if checked
- */
-:host {
-  --segment-button-color: var(
-    --ino-segment-button-color,
-    #{theme.color(light, base)}
-  );
-  --segment-button-checked-color: var(
-    --ino-segment-button-checked-color,
-    #{theme.color(primary, dark)}
-  );
-  --segment-button-hover-color: var(
-    --ino-segment-button-hover-color,
-    #{theme.color(primary, light)}
-  );
-  --segment-button-disabled-color: var(
-    --ino-segment-button-disabled-color,
-    #{theme.color(dark, base)}
-  );
-  --segment-button-hover-shadow: var(
-    --ino-segment-button-hover-shadow,
-    #{0 4px 8px 0 rgba(61, 64, 245, 0.3),
-    0 12px 28px 0 rgba(61, 214, 245, 0.14)}
-  );
-  --segment-button-checked-shadow: var(
-    --ino-segment-button-checked-shadow,
-    #{0 8px 16px 0 rgba(61, 64, 245, 0.3),
-    0 16px 32px 0 rgba(61, 214, 245, 0.14)}
-  );
+/* Colors: */
+$button_outlined_text_pressed: #d7e0ff;
+$button_outlined_text_hover: #e8edff;
+$text-active: #3b14f9;
+
+@mixin disabled {
+  pointer-events: none;
+  color: theme.$n-3;
 }
 
-:host {
+ino-segment-button {
   float: left;
 
-  .mdc-button {
+  .button {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 8px 16px;
+    border: none;
+    background-color: transparent;
+    color: theme.$n-3;
+
+    @include common.border-rounding-animation(
+      $round-border,
+      $edged-border,
+      0.15s,
+      $round-to-edged: true
+    );
+
+    min-height: 44px;
+    line-break: auto;
+    line-height: 100%;
+
+    cursor: pointer;
     @include typography.font($size: s, $weight: none);
```

Comment:
One thing left: the font should be medium according to the design. I think that should be `medium` instead of `none`.